websites.yml

```diff
@@ -46,3 +46,15 @@ username_patterns:
     invalid_patterns:
       - "\\.(com|net)"
       - "(\\.)\\1{1,}"  # consecutive '.' not allowed
+avatar:
+  pinterest:
+    property: image_xlarge_url
```

Comment:
is ``property`` a good term for this?

Reply:
how about `key` if you don't like it?

Reply:
i think ``key`` would be better since we're picking from ``json`` data. ``property`` seems vague here

Reply:
done.